pkgs/tools/security/pcsc-tools/default.nix

```diff
@@ -7,17 +7,22 @@
 , makeWrapper
 , pkg-config
 , wrapGAppsHook
-, systemd
-, dbus
+, systemdSupport ? lib.meta.availableOn stdenv.hostPlatform systemd, systemd
+, dbusSupport ? stdenv.isLinux, dbus
```

Comment:
`lib.meta.availableOn` also applies here, and within pcsclite.

Reply:
I don't think so. dbus is available on Darwin, but that doesn't mean it makes sense to build it for that platform. Though "if on Linux, enable dbus by default" is a good assumption.